public/app/core/components/Select/MetricSelect.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import _ from 'lodash';
+
+import Select from './Select';
+import { Variable } from 'app/types/templates';
+
+export interface Props {
+  onChange: (value: string) => void;
+  options: any[];
+  isSearchable: boolean;
+  value: string;
+  placeholder?: string;
+  className?: string;
+  groupName?: string;
+  variables?: Variable[];
+}
+
+interface State {
+  options: any[];
+}
+
+export class MetricSelect extends React.Component<Props, State> {
+  static defaultProps = {
+    variables: [],
+    options: [],
+    groupName: 'Options',
```

Comment:
add isSearchable: true would be a nice default (seems like all use cases except one wants searchable)

Reply:
👍